tests/tizen_capi/unittest_tizen_capi.cc

```diff
@@ -5535,6 +5535,172 @@ TEST (nnstreamer_capi_internal, validate_model_file_02_n)
   g_free (test_model2);
 }
 
+/**
+ * @brief Invoke callback for custom filter.
+ */
+static int
+test_custom_filter_invoke_cb (const ml_tensors_data_h in, ml_tensors_data_h out,
+    void *user_data)
+{
+  /* test code, set data size. */
+  if (user_data) {
+    void *raw_data = NULL;
+    size_t *data_size = (size_t *) user_data;
+
+    ml_tensors_data_get_tensor_data (out, 0, &raw_data, data_size);
+  }
+
+  return 0;
+}
+
+/**
+ * @brief Test for custom-easy registration.
+ */
+TEST (nnstreamer_capi_custom, register_filter_01_p)
+{
+  const char test_custom_filter[] = "test-custom-filter";
+  ml_pipeline_h pipe;
+  ml_pipeline_src_h src;
+  ml_pipeline_sink_h sink;
+  ml_custom_h custom;
```

Comment:
In my humble opinion, `ml_custom_h` could make users confused.
The framework `custom` of tensor_filter is different from `custom-easy`.
Because of this reason, what about using `ml_custom_eash_h` or other things?

Reply:
how about changing to ml_custom_filter_h?